repo/packages/D/dcos-ui/1/resource.json

```diff
@@ -0,0 +1,7 @@
+{
+  "assets": {
+    "uris": {
+      "dcos-ui-bundle": "https://downloads.mesosphere.com/dcos-ui/master%2Bdcos-ui-v2.24.4.tar.gz"
```

Comment:
Shall we go with `2.37.0`?

Reply:
Is `2.37.0` a "working" UI version? It hasn't been merged yet.

Reply:
We don't need to merge a version to `dcos` to have it on the Universe. That's the whole point 🎉

Reply:
Ah, ok.